### PR TITLE
chore: readme typos

### PR DESCRIPTION
**Pull request by dammy001**
## Change Summary

Provide summary of changes with issue number if any.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [x] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,47 +236,47 @@ Access Dashboard using : [http://localhost:8080/dashboard](http://localhost:8080
...
+# Production Setup
 
-By default, SQLite is used for storing meta data. However, you can specify your own database. The connection params for this database can be specified in `NC_DB` environment variable. Moreover, we also provide the below environment variables for configuration.
+By default, SQLite is used for storing metadata. However, you can specify your database. The connection parameters for this database can be specified in `NC_DB` environment variable. Moreover, we also provide the below environment variables for configuration.
 
-## Environment variables 
+## Environment variables
 
-Please refer to [Environment variables](https://docs.nocodb.com/getting-started/environment-variables)
+Please refer to the [Environment](https://docs.nocodb.com/getting-started/environment-variables)[ variables](https://docs.nocodb.com/getting-started/environment-variables)](https://docs.nocodb.com/getting-started/environment-variables)
```
Comment (wingkwong) [inline README.md:277]: broken

image

**Review by wingkwong — CHANGES_REQUESTED** [review]

**Comment by dammy001** [comment]
@wingkwong fixed

File: README.md [hunk 2]
```diff
@@ -182,7 +189,7 @@ docker-compose up -d
 
 # GUI
 
-Access Dashboard using : [http://localhost:8080/dashboard](http://localhost:8080/dashboard)
+Access Dashboard using: [http://localhost:8080/dashboard](http://localhost:8080/dashboard)](http://localhost:8080/dashboard)
```
Comment (wingkwong) [inline README.md:192]: broken here as well

**Review by wingkwong — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dammy001) [reply README.md]: resolved @wingkwong

**Review by wingkwong — APPROVED** [review]

**Pull request merged**